pkg/userdata/ubuntu/crio.go

```diff
@@ -7,6 +7,9 @@ func getCRIOInstallCandidate(desiredVersion string) (pkg string, version string,
 	case "1.9", "1.9.0":
 		pkg = "cri-o"
 		version = "1.9.0-1~ubuntu16.04.2~ppa1"
+	case "":
+		pkg = "cri-o"
+		version = "1.9.0-1~ubuntu16.04.2~ppa1"
```

Comment:
As mentioned in slack, we should have a dedicated logic which will find the best container-runtime version for the given kubernetes version.
But can be done in a separate pr as well. For now that is alright

Reply:
issue: #35